### PR TITLE
View component in studio test

### PR DESCRIPTION
**Pull request by Muddasser**

### CODE REVIEW

**Comment by Muddasser** [comment]
Please review this PR. 

This PR is primarily related to [LT-11](https://openedx.atlassian.net/browse/LT-11) but I have also done some refactoring in other tests. 

@raeeschachar @benpatterson

File: regression/pages/studio/course_outline_page.py [hunk 1]
```diff
@@ -46,22 +47,31 @@ def add_subsection_with_name(self, text):
...
         self.wait_for_ajax()
 
-    def add_unit(self):
+    def click_add_unit_button(self):
         """
         Adds Unit clicking the unit button of a sub section
         Navigates to Add Components page
         """
-        self.q(css='.button-new[data-default-name="Unit"]').click()
+        self.q(css='.button-new[data-default-name="Unit"]').results[-1].click()
```
Comment (benpatterson) [inline regression/pages/studio/course_outline_page.py]: why the `-1`?

File: regression/pages/studio/unit_page.py [hunk 2]
```diff
@@ -35,6 +44,9 @@ def add_word_cloud_component(self):
         self.wait_for_element_visibility(
             '.xblock-header-word_cloud', 'Word Cloud component displayed'
         )
+        if publish:
+            self.q(css='.action-publish.action-primary').click()
+            self.wait_for_ajax()
```
Comment (benpatterson) [inline regression/pages/studio/unit_page.py]: could you put in a comment on the ajax call?

File: regression/pages/studio/course_outline_page.py (hunk 1 above)
Comment (Muddasser) [inline regression/pages/studio/course_outline_page.py]: @benpatterson Its just a preventive measure. Section to which we add the unit appears at the end of page ( appended to the list of currently present sections) and we want to make sure we add unit to the subsection of this section.

**Comment by benpatterson** [comment]
:+1: feel free to merge from my perspective

**Comment by raeeschachar** [comment]
👍 Merge it!

**Pull request merged**